src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -671,6 +677,22 @@ public virtual EntityTracker EntityTracker
             }
         }
 
+        /// <summary>
+        /// Returns an instance of HttpClient.
+        /// </summary>
+        public virtual HttpClient HttpClient
+        {
+            get
+            {
+                return this.httpClient;
+            }
+
+            set
+            {
+                this.httpClient = null;
+            }

```

Comment:
Why not just have a get only prop instead?

Reply:
will people be able to pass their own instance if I make it only a get prop?

Reply:
ok got it